Makefile

```diff
@@ -397,11 +398,15 @@ deploy-test: _verify_cluster generate-keys
 
 # execute on the host, without skipper
 deploy-service-for-subsystem-test: create-hub-cluster load-image
-	skipper $(MAKE) deploy-test TARGET=kind SERVICE=$(SUBSYSTEM_LOCAL_SERVICE) IP=$(shell ip route get 1 | sed 's/^.*src \([^ ]*\).*$$/\1/;q')
+	skipper $(MAKE) deploy-test SERVICE=$(LOCAL_ASSISTED_SERVICE_IMAGE) IP=$(shell ip route get 1 | sed 's/^.*src \([^ ]*\).*$$/\1/;q')
```

Comment:
`IP=$(shell ip route get 1 | sed 's/^.*src \([^ ]*\).*$$/\1/;q')`
 if you trying to get the host IP, its better to use `hostname --ip-address` 

also, I prefer to avoid using expressions if I can
you can use this instead
```
ip route show  default  | cut -d" " -f9
```

Reply:
Appreciate the review, not part of this PR's changes though